packages/flutter/lib/src/material/drawer.dart

```diff
@@ -122,6 +122,7 @@ class DrawerControllerState extends State<DrawerController> {
 
   @override
   void dispose() {
+    _historyEntry?.remove();
```

Comment:
A side effect of this is that _handleHistoryEntryRemoved starts the close animation (calls close()). Since the controller is stopped by the next statement, that's probably harmless.